### PR TITLE
Introduce plugin for weekly reminder of open "Post-mortem" issues

### LINKED ISSUE #13: Scheduled reminder of unsolved post-mortem action items
We've been addressing action items for a couple of incidents now, solving a couple of them and leaving another couple to be forgotten forever. 

Since solving those action items is quite important to us, it would be useful to create a prow plugin that reminds us about all issues with a specific label representing a post-mortem action item.

This plugin would be enabled for:
* https://github.com/gitpod-io/gitpod
* https://github.com/gitpod-com/gitpod
* https://github.com/gitpod-com/observability

### PR DESCRIPTION
**Pull request by ArthurSens**
When ready, will fix #13. It has been almost half a year since I wrote any software, so bear with me on this probably slow implementation 😅.

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitbot/pull/19#" title="Author self-approved">ArthurSens</a>*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitbot).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/gitpod-io/gitbot/blob/main/OWNERS)~~ [ArthurSens]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: plugins/postmortem-reminder/go.mod [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module github.com/gitpod-io/gitbot/postmortem-reminder
```
Comment (leodido) [inline plugins/postmortem-reminder/go.mod:1]: Don't forget to run `go mod tidy` :)

**Review by leodido — COMMENTED** [review]
I know it's a WIP.

Just wanted to leave a nit here :)

File: plugins/postmortem-reminder/go.mod (hunk 1 above)
Comment (leodido) [reply plugins/postmortem-reminder/go.mod:1]: BTW, it's lovely you're writing your first Prow plugin!

I feel more will follow :)

Comment (ArthurSens) [reply plugins/postmortem-reminder/go.mod:1]: > Don't forget to run `go mod tidy` :)

For some reason `go mod tidy` doesn't do anything for me 🤔. Well, actually it fails with:
```
github.com/gitpod-io/gitbot/postmortem-reminder imports
        k8s.io/test-infra/prow/flagutil imports
        sigs.k8s.io/controller-runtime/pkg/client imports
        k8s.io/client-go/metadata: package k8s.io/client-go/metadata provided by k8s.io/client-go at latest version v0.21.3 but not at required version v11.0.1-0.20190805182717-6502b5e7b1b5+incompatible
github.com/gitpod-io/gitbot/postmortem-reminder imports
        k8s.io/test-infra/prow/flagutil imports
        k8s.io/client-go/kubernetes imports
        k8s.io/client-go/discovery imports
        github.com/googleapis/gnostic/OpenAPIv2: module github.com/googleapis/gnostic@latest found (v0.5.5), but does not contain package github.com/googleapis/gnostic/OpenAPIv2
github.com/gitpod-io/gitbot/postmortem-reminder imports
        k8s.io/test-infra/prow/flagutil imports
        k8s.io/client-go/kubernetes imports
        k8s.io/client-go/kubernetes/typed/auditregistration/v1alpha1 imports
        k8s.io/api/auditregistration/v1alpha1: module k8s.io/api@latest found (v0.21.3), but does not contain package k8s.io/api/auditregistration/v1alpha1
github.com/gitpod-io/gitbot/postmortem-reminder imports
        k8s.io/test-infra/prow/flagutil imports
        k8s.io/client-go/kubernetes imports
        k8s.io/client-go/kubernetes/typed/batch/v2alpha1 imports
        k8s.io/api/batch/v2alpha1: module k8s.io/api@latest found (v0.21.3), but does not contain package k8s.io/api/batch/v2alpha1
github.com/gitpod-io/gitbot/postmortem-reminder imports
        k8s.io/test-infra/prow/flagutil imports
        k8s.io/client-go/kubernetes imports
        k8s.io/client-go/kubernetes/typed/settings/v1alpha1 imports
        k8s.io/api/settings/v1alpha1: module k8s.io/api@latest found (v0.21.3), but does not contain package k8s.io/api/settings/v1alpha1
```

But I couldn't figure out how to solve that

Comment (csweichel) [reply plugins/postmortem-reminder/go.mod:1]: You're missing the `replace` rule for k8s.io in your `go.mod`. Check out the `go.mod` of the other plugins to see what I mean

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **arthursens** after the PR has been reviewed.

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitbot).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitbot/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request closed without merging**